students/dowoon/users/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+class User(models.Model):
+    name = models.CharField(max_length=25)
+    email = models.CharField(max_length=50)
+    password = models.CharField(max_length=25)
+    phone = models.CharField(max_length=25)
```

Comment:
가독성을 위해 = 혹은 :를 기준으로 정렬해주세요!
```suggestion
    name     = models.CharField(max_length=25)
    email    = models.CharField(max_length=50)
    password = models.CharField(max_length=25)
    phone    = models.CharField(max_length=25)
```